web/src/common/modal/CaptchaModal.js

```diff
@@ -115,7 +115,7 @@ export const CaptchaModal = (props) => {
     } else {
       return (
         <Col>
-          <Row>
+          <Row style={{justifyContent: "center"}}>
```

Comment:
Perhaps we can refer to ` https://ant-design.antgroup.com/components/grid#row` for style settings

Reply:
I will modify the implementation method in the code.